src/explorer.ts

```diff
@@ -300,6 +323,10 @@ export class OpenShiftExplorer implements TreeDataProvider<ExplorerItem>, Dispos
                     result = [...helmRepos.sort(Helm.ascRepoName)];
                 }
             }
+        } else if ('kind' in element && element.kind === 'Deployment') {
+            const pods = await Oc.Instance.getKubernetesObjects('pods');
+            const filteredPods: DeploymentPodObject[] = pods.filter((pod) => pod.metadata.name.indexOf(element.metadata.name) !== -1 && pod);
```

Comment:
I don't think `&& pod` is needed here. `&& pod` will check if pod is not `null` or `undefined`, but since pod has a field `metadata` that you access, we know it's not null.

Reply:
Removed `pod`